api/src/main/resources/application.properties

```diff
@@ -18,8 +18,8 @@ spring.flyway.enabled=true
 spring.flyway.validate-on-migrate=true
 spring.flyway.baseline-on-migrate=true
 spring.flyway.schemas=public
-spring.flyway.url=jdbc:postgresql://localhost:5432/amazinDb
+spring.flyway.url=jdbc:postgresql://localhost:5432/amazinDB
 spring.flyway.user=postgres
-spring.flyway.password=123456
+spring.flyway.password=root
```

Comment:
See above comment... however this is more important now since flyway also has these settings in `pom.xml` file. If we change it here we need to change it there as well. (I was hedging my bets due to flyway connection issues).